Makefile

```diff
@@ -58,7 +68,7 @@ perf-test-run: perf-test-build
 perf-test-help: perf-test-build
 	go run perfTest/perftest.go help
 
-perf-test-build: vet fmt check
+perf-test-build:
```

Comment:
Removing those pre-steps because `vet`, `fmt` and `check` are run as part of `test`. It is unnecessary to run them again in the build step.